README.md

```diff
@@ -60,12 +72,12 @@ $ cargo build --release
 ## Usage
 
 In the following sections we'll be using excerpts from the
-[examples](./examples/rust-example). You can actually run the complete
+[C examples](./examples/c-example). You can actually run the complete
 example using Cargo from inside that directory.
 
 ```shell
-$ cd examples/rust-example
-$ cargo test
+$ cd ../examples/c-example
+$ make test
```

Comment:
This is a dumb question, but just to double check, you ran this and it works?

Reply:
:+1: Yes

Reply:
Assuming this is still coming from the perspective of the root README, should we loose the `../` part of the `cd`?